### PR TITLE
Home page with footer

### LINKED ISSUE #3: home page with masonry
Home

### PR DESCRIPTION
**Pull request by harshil1712**
Fixes #3 and #1 
here is the output

Chrome
Screenshot from 2019-06-27 07-02-05

Firefox
Screenshot from 2019-06-27 07-10-26

### CODE REVIEW

**Comment by obitodarky** [comment]
I think we can use the styled button component in a different file. The header also uses that button and it's better to separate styled components for clean code.

File: src/pages/index.js [hunk 1]
```diff
@@ -1,21 +1,91 @@
...
+            <Typography variant="inherit" style={{ fontFamily: "Zilla Slab" }}>
+              We{" "}
+              <span role="img" aria-label="heart emoji">
+                &#x2764;&#xFE0F;
+              </span>{" "}
+              the Web!
+            </Typography>
+          </div>
+        </GridList>
+        <StyledButton>
```
Comment (obitodarky) [inline src/pages/index.js]: I tried to implement the styled button in my header. It appears fine but it isn't clickable for me. Have you checked if you can perform actions on this StyledButton?

**Comment by harshil1712** [comment]
@obitodarky I agree. I was planning to do that once we are ready with few pages. We tried the component first approach it only made us slow. We can do that once we have the MVP ready.

**Review by obitodarky — COMMENTED** [review]
@harshil1712 if you can make a new component and also check if the styled button is clickable then I would like to merge this to master and then work on the header.

**Comment by harshil1712** [comment]
Since I haven't added any on press methods nor linked to a different page,
the button isn't working. But if we add a method or a link it will work

**Pull request closed without merging**